src/preset_cli/cli/superset/sync/dbt/command.py

```diff
@@ -338,7 +348,44 @@ def get_job(
     raise ValueError(f"Job {job_id} not available")
 
 
-def process_sl_metrics(
+def get_sl_metric(
+    metric: Dict[str, Any],
+    model_map: Dict[ModelKey, ModelSchema],
+    dialect: MFSQLEngine,
+) -> Optional[MFMetricWithSQLSchema]:
+    """
+    Compute a SL metric using the ``mf`` CLI.
+    """
+    mf_metric_schema = MFMetricWithSQLSchema()
+
+    command = ["mf", "query", "--explain", "--metrics", metric["name"]]
+    try:
+        result = subprocess.run(command, capture_output=True, text=True, check=True)
+    except subprocess.CalledProcessError:
+        return None
```

Comment:
Now the CLI requires having `metricflow` installed to properly retrieve the SQL syntax for metrics. I wonder if we can identify the error returned in case it's not installed (maybe something like `command not found` and provide an error to the user instructing to install the package. We could use `sys.exit(1)` to make sure a CI job would also show as failed so the user adds the specific `metricflow` package (according to their engine) to their configurations.

We could also add the package to the list of requirements.